src/main/java/frc/robot/subsystems/interfaces/TimeOfFlightInterface.java

```diff
@@ -0,0 +1,16 @@
+package main.java.frc.robot.subsystems.interfaces;
```

Comment:
I dont think we want this subsystems. the time of flight sensor is used by the magazine subsystem, but probably doesnt warrant its own subsystem. Is there are reasons why we need to manage this as a standalone system?